pytun.py

```diff
@@ -110,8 +110,13 @@ def main():
         test_tunnels_and_exit(files, logger, processes)
 
     if args.test_all:
+        http_inspection = inspection_http_server(tunnel_path, tunnel_manager_id, LogManager.path, Status(), __version__,
```

Comment:
si el servicio esta levantado no vas a poder levantar el thread y no deberias de hacerlo

Reply:
Pero lo levanto despues, 
`    if not service_up:
        logger.info("The service is not running! You won't be able to access your services from the cloud")
        if introspection_thread:
            introspection_thread.start()`